packages/react-native-usb/android/src/main/java/io/trezor/rnusb/ReactNativeUsbModule.kt

```diff
@@ -234,15 +234,17 @@ class ReactNativeUsbModule : Module() {
 
     private fun closeDevice(deviceName: String) {
         Log.d(LOG_TAG, "Closing device $deviceName")
-        val usbConnection = getOpenedConnection(deviceName)
-        usbConnection.close()
+        getOpenedConnection(deviceName).close()
```

Comment:
Does this really solves the issue? It doesn't seems like anything that should affect functionallity.

Reply:
No, the important change is in `closeAllOpenedDevices()`. The variable here is just redundant.